spin/command.go

```diff
@@ -19,7 +19,16 @@ func (o Options) Run() error {
 
 	s := spinner.New()
 	s.Style = o.SpinnerStyle.ToLipgloss()
-	s.Spinner = spinnerMap[o.Spinner]
+
+	if len(o.Glyphs) > 0 {
+		s.Spinner = spinner.Spinner{
+			Frames: o.Glyphs,
+			FPS:    o.FrameInterval,
+		}
+	} else {
+		s.Spinner = spinnerMap[o.Spinner]
+	}
```

Comment:
I tend to avoid else when possible

```suggestion
	s.Spinner = spinnerMap[o.Spinner]
	if len(o.Glyphs) > 0 {
		s.Spinner = spinner.Spinner{
			Frames: o.Glyphs,
			FPS:    o.FrameInterval,
		}
	}
```